benchmaster/benchmaster.py

```diff
@@ -96,6 +102,10 @@
     --sibench-workers FACTOR          Workers per server = factor x no of cores.                sweepable  [default: 1.0]
     --sibench-skip-read-verification  Disable read validation for speed.    
     --sibench-root-password PW        Root password for the sibench servers                                [default: linux]
+    --sibench-generator GEN           Workload generator for sibench                                       [default: prng]
+    --sibench-slice-dir DIR           Directory for the corpus if using the slice generator
```

Comment:
given `--sibench-slice-dir` doesn't have a default, should we try to catch if this option is not being provided when the generator is `slice`, to fail gracefully?

Reply:
Or just stick in a default?

Reply:
Also, try to change the author/commiter name for this branch (currently **root**)